app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -177,6 +183,12 @@ class HomeFragment : Fragment() {
                 requireComponents.analytics.metrics.track(Event.OpenedAppFirstRun)
             }
         }
+
+        if (!onboarding.userHasBeenOnboarded() &&
+            requireContext().settings().shouldShowPrivacyPopWindow &&
+            Config.channel.isMozillaOnline) {
+            showPrivacyPopWindow()
```

Comment:
Instead of creating a `webview` I think you can take advantage of the whole browser and load the links directly on it by using [openToBrowserAndLoad](https://github.com/mozilla-mobile/fenix/blob/d8f3127e0868c80e0e782ac6fab11462f5fbbc96/app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt#L808) as we do to load other web content :)

Reply:
Thanks! I've tried openToBrowserAndLoad but there are still two problems:
1. Clicking the hyper links make the tab opened behind the alert dialog:
![Screenshot_20201216_095416_org mozilla fenix debu](https://user-images.githubusercontent.com/33616048/102294733-ea0ebf00-3f84-11eb-8eb3-43d05b072768.jpg)
2. If starting a new activity to open the tabs to avoid the alert dialog's shelter, users may change the url in the address bar to use fenix directly, which means they can use fenix without approving the privacy notice.

Reply:
Fenix supports [custom tabs](https://developer.chrome.com/docs/multidevice/android/customtabs/), talking with @jonalmeida he shared this [example](https://github.com/mozilla-mobile/fenix/blob/master/app/src/main/java/org/mozilla/fenix/settings/account/AuthIntentReceiverActivity.kt#L29-L34) where we used it from Fenix. Worst case scenario if custom tabs are not OK for your use case, you can embed geckoview see an example [here](https://github.com/mozilla-mobile/fenix/blob/master/app/src/main/res/layout/fragment_add_on_internal_settings.xml)

Reply:
Thanks, I will try it and update this pr.